### PR TITLE
#1850 Split of upgrader (with/without helm upgrade), update of release notes

### PR DESCRIPTION
**Pull request by johannes-b**
- Split of upgrader into two versions: (1) converts all Helm release from v2 to v3, (2) does not convert Helm releases.  
- Added note (that two upgraders are available) to release notes

### CODE REVIEW

File: releasenotes/releasenotes_V0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Fixed Issues
+
+
+## Development Process
+
+
+## Good to know / Known Limitations
+
+* **Upgrade from 0.6.2 to 0.7:** *Keptn 0.7 uses Helm 3 while previous Keptn releases rely on Helm 2*. To upgrade  your Helm releases from Helm 2 to 3, two options are provided: 
+  1. *Job without Helm 3 Upgrade:* This option is needed when the cluster contains Helm releases not managed by Keptn. If this job is executed, it is necessary to manually converted the releases from Helm 2 to 3 as explained on [keptn.sh/docs](http://localhost:1313/docs/0.7.0/operate/upgrade/#job-without-helm-3-0-upgrade).
```
Comment (agrimmer) [inline releasenotes/releasenotes_V0.7.0.md:32]: Please update the link (i.e. contains localhost).

File: releasenotes/releasenotes_V0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  1. *Job with Helm 3 Upgrade:* Full automation of Helm upgrade for installations were just Keptn is installed. If this job is executed, **all** Helm releases on the cluster are converted from Helm 2 to 3.
```
Comment (agrimmer) [inline releasenotes/releasenotes_V0.7.0.md:33]: Please add "and Tiller is removed".

File: releasenotes/releasenotes_V0.7.0.md (hunk 1 above)
Comment (johannes-b) [reply releasenotes/releasenotes_V0.7.0.md]: Good catch! Fixed it.

File: releasenotes/releasenotes_V0.7.0.md (hunk 2 above)
Comment (johannes-b) [reply releasenotes/releasenotes_V0.7.0.md]: I added the note.

**Pull request merged**